[2023-03-24T21:16:00.982+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:15:00+00:00 [queued]>
[2023-03-24T21:16:00.991+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:15:00+00:00 [queued]>
[2023-03-24T21:16:00.991+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-24T21:16:00.991+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-03-24T21:16:00.991+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-24T21:16:01.005+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): print_date> on 2023-03-24 21:15:00+00:00
[2023-03-24T21:16:01.009+0000] {standard_task_runner.py:55} INFO - Started process 3322 to run task
[2023-03-24T21:16:01.012+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test-tutorial', 'print_date', 'scheduled__2023-03-24T21:15:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/test-tutorial.py', '--cfg-path', '/tmp/tmpf6l1pwa6']
[2023-03-24T21:16:01.012+0000] {standard_task_runner.py:83} INFO - Job 53: Subtask print_date
[2023-03-24T21:16:01.072+0000] {task_command.py:388} INFO - Running <TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:15:00+00:00 [running]> on host e7c16d9fd17d
[2023-03-24T21:16:01.142+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=daniel@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test-tutorial
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2023-03-24T21:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-24T21:15:00+00:00
[2023-03-24T21:16:01.143+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-24T21:16:01.143+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date +%Y-%m-%d %H:%M:%S']
[2023-03-24T21:16:01.153+0000] {subprocess.py:86} INFO - Output:
[2023-03-24T21:16:01.156+0000] {subprocess.py:93} INFO - date: extra operand ‘%H:%M:%S’
[2023-03-24T21:16:01.156+0000] {subprocess.py:93} INFO - Try 'date --help' for more information.
[2023-03-24T21:16:01.156+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-03-24T21:16:01.168+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-03-24T21:16:01.173+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=test-tutorial, task_id=print_date, execution_date=20230324T211500, start_date=20230324T211600, end_date=20230324T211601
[2023-03-24T21:16:01.185+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 53 for task print_date (Bash command failed. The command returned a non-zero exit code 1.; 3322)
[2023-03-24T21:16:01.223+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-24T21:16:01.254+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
